lib/config-validator.ts

```diff
@@ -71,9 +71,9 @@ type PackageJson = {
     }
   }
   try {
-    const pkgJson = JSON.parse(
-      readFileSync('package.json', 'utf8')
-    ) as PackageJson;
+    const pkgJson = (await JSON.parse(
```

Comment:
One await to much. The parse isn't async. 😉

Reply:
Oops, not sure how I got that there twice :smile: thanks, fixed.